lab_5_scrapper/scrapper.py

```diff
@@ -116,6 +221,9 @@ def __init__(self, config: Config) -> None:
         Args:
             config (Config): Configuration
         """
+        self.config = config
+        self.urls = []
+        self.url_pattern = 'https://new-science.ru'
```

Comment:
is it possible to extract this pattern from one of your seed urls during initialization?

Reply:
it is possible, done